pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -324,6 +328,12 @@ type operationExecutor struct {
 	// pendingOperations keeps track of pending attach and detach operations so
 	// multiple operations are not started on the same volume
 	pendingOperations goroutinemap.GoRoutineMap
+
+	// recorder is used to pass events
+	recorder record.EventRecorder
+
+	//pod ref
+	PodRef VolumeToMount
```

Comment:
yeah, since all I'm doing is bringing in the VolumeToMount struct to be used by the oe, I agree, I will reword and see if it makes more sense